test/new-e2e/tests/containers/k8s_test.go

```diff
@@ -1006,12 +1006,12 @@ func (suite *k8sSuite) testAdmissionControllerPod(namespace string, name string,
 			}
 		}
 
-		if suite.Contains(env, "PYTHONPATH") {
-			suite.Equal("/datadog-lib/", env["PYTHONPATH"])
-		}
```

Comment:
I do't understand this change but I'm going to merge to unblock the CI. We'll investigate with the teams

Reply:
The test code itself was a bit weird because it's doing an `if suite.Contains(...)` but `suite.Contains` is  also an assertion, so if the env vars didn't contain `PYTHONPATH` we fail.


This is equivalent to:

```go
suite.Contains(env, "PYTHONPATH")
suite.Equal("/datadog-lib/", env["PYTHONPATH"])
```

#27811 changed the behavior of the auto-instrumentation webhook to no longer put in the environment variable for the language (in this case python) into the envs of the container -- this is true for all of the languages. What happens now is we inject:

```
"LD_PRELOAD":"/opt/datadog-packages/datadog-apm-inject/stable/inject/launcher.preload.so"
"DD_INJECT_SENDER_TYPE":"k8s",
"DD_INJECT_START_TIME":"<timestamp>"
```